### ISSUE TITLE
riak-debug relative data directory path crash

### ISSUE THREAD

**Issue by binarytemple** [basho/riak#604]
Running Riak from `make devrel`...

The storage has been configured with mult, and there is a leveldb datastore named `big_leveldb`.

The `big_leveldb` datastore has been configured to use `data/big_leveldb` and Riak is fine with this, loading data in and out to the associated bucket-type.. 

However riak-debug is not, it crashes with the following error:

```
[/basho/riak/dev/dev1%]bin/riak-debug
........E......EE..............Unable to locate <<"big_leveldb">> LevelDB data directory. Aborting.
Using riak_kv_eleveldb_backend data_root: data/big_leveldb
```

The problem goes away when an absolute path is given.

**Comment by pyrrho** [comment]
Two pull requests that relate to this, #676 will fix this bug in the 2.0 branch and #678 will fix it in the 1.4 brach.

**Comment by pyrrho** [comment]
As the above PRs have already been merged, I'm going to go ahead and close this issue. Feel free to re-open if there's more to discuss.
